sound/soc/sof/amd/acp.h

```diff
@@ -82,6 +83,8 @@
 #define SRAM1_SIZE				0x13A000
 #define PROBE_STATUS_BIT			BIT(31)
 
+#define FIRMWARE_SIGNATURE			0x100
```

Comment:
prefix?

Reply:
Okay, Thanks : )
We will change it  to ACP_FIRMWARE_SIGNATURE.